docker-compose.yml

```diff
@@ -0,0 +1,143 @@
+version: "3"
+
+services:
+  pinning_service-worker:
+    image: python:3.9
+    volumes: 
+      - .:/app
+    entrypoint: /bin/sh
+    command:
+      - -c
+      - | 
+        pip install poetry
+        cd /app
+        poetry config virtualenvs.create false 
+        poetry install --no-interaction --no-ansi
+        make task-queue-init
+        make task-queue-worker
+    environment:
+      POSTGRES_SERVER: "postgres"
+      POSTGRES_USER: "postgres"
+      POSTGRES_PASSWORD: "postgres"
+      POSTGRES_DB: "app"
+      # Regen node
+      REGEN_NODE_REST_URL: "http://localhost:1317"
+      REGEN_NODE_TENDERMINT_RPC_URL: "http://localhost:26657"
+      # TX Signing
+      REGEN_RESOLVER_ID: 1
+      REGEN_CLI_COMMAND: "docker run --rm -i --network host -v $(pwd)/localnet/keys:/regen-keys ghcr.io/cambiumteam/regen-cli:latest"
+      REGEN_KEYRING_ARGS: "--keyring-backend test --keyring-dir /regen-keys/"
+      REGEN_CHAIN_ID: "regen-1"
+      REGEN_KEY_ADDRESS: ""
+      REGEN_GAS_PRICES_AMOUNT: 6e-6
+      REGEN_GAS_PRICES_DENOM: "stake"
+  pinning_service-api:
+      image: python:3.9
+      ports: 
+        - "8000:8000"
+      volumes: 
+        - .:/app
+      entrypoint: /bin/sh
+      command:
+        - -c
+        - | 
+          pip install poetry
+          pip install uvicorn
+          cd /app
+          poetry config virtualenvs.create false 
+          poetry install --no-interaction --no-ansi
+          make start
+      environment:
+        POSTGRES_SERVER: "postgres"
+        POSTGRES_USER: "postgres"
+        POSTGRES_PASSWORD: "postgres"
+        POSTGRES_DB: "app"
+        # Regen node
+        REGEN_NODE_REST_URL: "http://localhost:1317"
+        REGEN_NODE_TENDERMINT_RPC_URL: "http://localhost:26657"
+        # TX Signing
+        REGEN_RESOLVER_ID: 1
+        REGEN_CLI_COMMAND: "docker run --rm -i --network host -v $(pwd)/localnet/keys:/regen-keys ghcr.io/cambiumteam/regen-cli:latest"
+        REGEN_KEYRING_ARGS: "--keyring-backend test --keyring-dir /regen-keys/"
+        REGEN_CHAIN_ID: "regen-1"
+        REGEN_KEY_ADDRESS: ""
+        REGEN_GAS_PRICES_AMOUNT: 6e-6
+        REGEN_GAS_PRICES_DENOM: "stake"
+  indexer:
```

Comment:
Specifically for the indexer, could we have the `restart: always` flag here? It would probably be smart on all of the services but the indexer stands out since it has the websocket connection which is prone to disconnect from time to time.